RockWeb/Themes/NewSpring/Scripts/get-distance.js

```diff
@@ -113,7 +115,7 @@ document.addEventListener("DOMContentLoaded", function() {
             var anchor = document.querySelector('#locations');
 
             // Scroll constructor located in _smooth-scroll.js, compiled into main.js
-            scroll.animateScroll(anchor);
+            // scroll.animateScroll(anchor);
```

Comment:
Not a big deal, but if you're going to remove it, might as well just remove it.

Reply:
@richarddubay i'm coming back to it, so that's why I left it for now - but you're right. If it was finished we would remove it.